config/config-tracing.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2023 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-tracing
+  namespace: tekton-pipelines
+  labels:
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-pipelines
+data:
+  _example: |
+    ################################
+    #                              #
+    #    EXAMPLE CONFIGURATION     #
+    #                              #
+    ################################
+    # This block is not actually functional configuration,
+    # but serves to illustrate the available configuration
+    # options and document them in a way that is accessible
+    # to users that `kubectl edit` this config map.
+    #
+    # These sample configuration options may be copied out of
+    # this example block and unindented to be in the data block
+    # to actually change the configuration.
+    #
+    # Enable sending traces to defined endpoint by setting this to true
+    enabled: "true"
+    #
+    # API endpoint to send the traces to
+    # (optional): The default value is given below
+    endpoint: "http://jaeger-collector.jaeger.svc.cluster.local:14268/api/traces"
```

Comment:
would it make sense to either:
- assume that if someone has configured an endpoint, that they want tracing enabled, i.e. remove the "enabled" setting
- make this configuration real configuration instead of an example configuration, and set this to be the actual default endpoint and set enabled to "false" by default
?

Reply:
> assume that if someone has configured an endpoint, that they want tracing enabled, i.e. remove the "enabled" setting

We will add more config options later. So it makes sense to make the `enable` flag explicit rather than based on the endpoint, which will be confusing to the users.

> make this configuration real configuration instead of an example configuration, and set this to be the actual default endpoint and set enabled to "false" by default

We follow this pattern of giving only an example for other configmaps like events and observability. It is better to keep it consistent